docs/protocol/stability/index.md

```diff
@@ -12,6 +12,13 @@ import PageRef from '@site/src/components/PageRef';
 
 Overview of the Celo protocol's Stability Mechanisms.
```

Comment:
honestly this page id just remove completely Its talking about mento

Reply:
removed from the menu/sidebar, kept the pages around for now to avoid breakage